vizro-core/src/vizro/static/js/models/date_picker.js

```diff
@@ -7,3 +7,13 @@ export function _update_date_picker_values(value, input_store) {
   }
   return [value, value];
 }
+
+export function _update_date_picker_position(clicks) {
```

Comment:
Where do we stand regarding the tests of our JS functions? We tested the framework, but haven't gotten to the implementation yet right?

Reply:
Only for a few client-side JS functions we implemented JS unit tests. We had a separate ticket for JS unit tests implementation that I can't find now, so I'll contact Joe.